Content/paket.dependencies

```diff
@@ -18,9 +18,9 @@ group Server
 //#elseif (!remoting && server == "suave")
     nuget Thoth.Json.Net
 //#elseif (remoting && server == "suave")
-    nuget Fable.Remoting.Suave
+    nuget Fable.Remoting.Suave ~> 3
```

Comment:
urgs explicit version ranges!?

Reply:
That makes it easier to update lock files without getting breaking changes (as long as the deps follow semantic versioning)